.travis.yml

```diff
@@ -66,15 +66,14 @@ script:
   - bundle exec rails db:schema:load
   - bundle exec rails assets:precompile
   - './cc-test-reporter before-build'
-  - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "0" ]; then yarn test --colors; fi
+  - if [ "$KNAPSACK_PRO_CI_NODE_INDEX" == "0" ]; then yarn test --colors;yarn e2e:ci; fi
```

Comment:
It's mentioned in a comment in `bin/e2e`, but after the engineering roundtable today, we we decided it was fine not to worry about cypress-rails/knapsack pro integration for the time being which is why e2e tests are explicitly running on the first node alongside front0end tests.